### PR TITLE
adding a recipe for wntr

### PR DESCRIPTION
**Pull request by michaelbynum**
This is my first recipe. Hopefully it works.

### CODE REVIEW

File: recipes/wntr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+    - pip
+  host:
+    - python
+    - pip
```
Comment (marcelotrevisani) [inline recipes/wntr/meta.yaml]: please, remove ``pip`` from here
I just saw that this project depends of anothers lobraries such as:
networkx, pandas, numpy, scipy, matplotlib, plotly

So, you have to add them to the host requirements and also to the run requirements

Comment (michaelbynum) [reply recipes/wntr/meta.yaml]: @marcelotrevisani Ah, thanks! Sorry - I am new to this. I will fix it now.

File: recipes/wntr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  url: https://github.com/michaelbynum/WNTR/archive/3c8a669d50e77e872e9e6dd2c391a9fa83c34ecd.zip
+  sha256: 5d33a18f7cee0f76ca7a99074243a5019ae70f76ccb84dc44be54106eb758589
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (marcelotrevisani) [inline recipes/wntr/meta.yaml]: is this project compiling C++ code?

File: recipes/wntr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  host:
+    - python
+    - pip
+    - numpy
+    - networkx
+    - pandas
+    - scipy
+    - matplotlib
+    - plotly
+    - nose
```
Comment (marcelotrevisani) [inline recipes/wntr/meta.yaml]: do you really need ``nose`` to build the package or just to test it?
If is just to test it you can remove ``nose`` from the build requirements and also from host

File: recipes/wntr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - python
+    - pip
```
Comment (marcelotrevisani) [inline recipes/wntr/meta.yaml]: ``pip`` has to be a build requirement, please change :)

File: recipes/wntr/meta.yaml (hunk 2 above)
Comment (michaelbynum) [reply recipes/wntr/meta.yaml]: @marcelotrevisani Yes, it is.

File: recipes/wntr/meta.yaml (hunk 3 above)
Comment (michaelbynum) [reply recipes/wntr/meta.yaml]: Okay, thank. I will remove it.

File: recipes/wntr/meta.yaml (hunk 4 above)
Comment (michaelbynum) [reply recipes/wntr/meta.yaml]: But not a host requirement?

Comment (marcelotrevisani) [reply recipes/wntr/meta.yaml]: Just for ``build``, please see
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#requirements-section

:)

File: recipes/wntr/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/wntr/meta.yaml]: No problem! We are here to learn  :)

**Comment by michaelbynum** [comment]
@marcelotrevisani Do you understand this error from CircleCI? I don't know what I would set the PYTHON environment variable to.

```No files or script found for output wntr
WARNING:conda_build.build:No files or script found for output wntr
Traceback (most recent call last):
  File "/home/conda/.ci_support/build_all.py", line 83, in <module>
    build_all(args.recipes_dir, args.arch)
  File "/home/conda/.ci_support/build_all.py", line 73, in build_all
    conda_build.api.build(list(d.keys()), config=config)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/api.py", line 200, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 2189, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 1575, in build
    built_package = bundlers[output_d.get('type', 'conda')](output_d, m, env, stats)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 921, in bundle_conda
    "run that executable.".format(dep, env_var_name))
conda_build.exceptions.CondaBuildException: Empty package; python present in build and host deps.  You probably picked up the build environment's python  executable.  You need to alter your recipe to  use the PYTHON env var in your recipe to run that executable.
Exited with code 1```

**Comment by marcelotrevisani** [comment]
hummm....
I think you have to put the build just the ``compiler`` and the others thing in ``host``

File: recipes/wntr/meta.yaml (hunk 4 above)
Comment (michaelbynum) [reply recipes/wntr/meta.yaml]: @marcelotrevisani Thanks. Should the meta.yaml example be updated? It has pip as a host requirement.

**Comment by michaelbynum** [comment]
I am going to close this pull request until I can figure out the empty package error.

**Pull request closed without merging**